react-stopwatch-component/src/index.jsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+class StopWatch extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleClick = this.handleClick.bind(this);
+    this.state = { started: false };
+    this.seconds = 0;
+    this.tick = this.tick.bind(this);
+    this.handlePauseClick = this.handlePauseClick.bind(this);
+    this.resetClock = this.resetClock.bind(this);
+  }
+
+  handleClick() {
+    this.setState({ started: true });
+    this.timerId = setInterval(() => this.tick(), 1000);
+  }
+
+  handlePauseClick() {
+    this.setState({ started: false });
+    clearInterval(this.timerId);
+  }
+
+  tick() {
+    this.seconds++;
+    const element = (
+      <div id="time">{this.seconds}</div>
+    );
+    ReactDOM.render(element, document.getElementById('circle'));
```

Comment:
Your component should be defined in `stopwatch.jsx` and you should only use `ReactDOM.render` in once place, in `index.jsx`